src/Server/Player/PlayerLogic.py

```diff
@@ -38,10 +39,11 @@ def __init__(self, player_id, terrain, game):
         self._vel = Vector2D.zero()
         self._forces = []
         self._is_flying = True
-        self._can_attack = True
+        self._attack_on_cooldown = False
         self._is_attacking = False
         self.can_get_hurt = True
-        self.hp = 100
+        self.max_hp = 100
+        self.hp = self.max_hp
 
     def update(self):
```

Comment:
That max_hp sounds like a constant. Poor him is crying to be treated correspondingly.

Reply:
Python somewhy hates those big letters :( apart from that if someone drinks super sugar juice he may increase his full hp :)